src/Components/DashBoard/Repository/RepoComponents/RepoDetails/RepoDetailBackdrop/RemoteConfigComponent/RemoteManagementComponent.js

```diff
@@ -35,15 +35,15 @@ export default function AddRemoteRepoComponent({ repoId }) {
       },
     })
       .then((res) => {
-        res.data.data.forEach((items) => {
+        res.data.data.remoteDetails.forEach((items) => {
           remoteArray.push(items);
         });
         setRemoteDetails([...remoteArray]);
         setStatusCheck(false);
       })
       .catch(() => {
         setStatusCheck(true);
-        setRemoteOperation("get");
+        setRemoteOperation("load");
```

Comment:
instead of tracking the request loading state in the same state variable tracking the operations, use a different one